packages/react/src/primitives/PasswordField/ShowPasswordButton.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { ShowPasswordButtonProps } from '../types/';
+import { Button } from '../Button';
+import { ComponentClassNames } from '../shared/constants';
+import { IconVisibility } from '../Icon';
+import { Text } from '../Text';
+import classNames from 'classnames';
+
+export const ShowPasswordButton: React.FC<ShowPasswordButtonProps> = ({
+  onClick = () => {},
```

Comment:
Should we use React's `MouseEventHandler` type here?

Reply:
I don't think this default value is needed, so I've removed it. The `onClick` prop is already typed as `MouseEventHandler`